.github/actions/get-releases/action.yml

```diff
@@ -38,10 +38,11 @@ runs:
         or indirectly use them
       shell: bash
       run: |
+        PACKAGE_FILTER_ONLY_NAME="$(echo '${{ inputs.package-filter }}'| sed -E 's/^[\.\^]+//' | sed -E 's/[\.\^]+$//')"
         pnpm \
           --filter="@repo/citools..." \
           --filter="@repo/release-it-base-isolated..." \
-          --filter="...${{ inputs.package-filter }}..." \
+          --filter="...${PACKAGE_FILTER_ONLY_NAME}..." \
```

Comment:
Copied from internal fork after we found that passing a filter with `...` in the trigger release workflow will break it